### PR TITLE
Adds Cosmohonk suit.

### PR DESCRIPTION
**Pull request by Zuvassin**
ss 2016-04-27 at 03 05 14

Credits to Peble for the sprite.

### CODE REVIEW

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -236,6 +236,21 @@ Contains:
 	flash_protect = 0
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 20)
 
+/obj/item/clothing/suit/space/eva/clown
+	name = "cosmohonk suit"
+	icon_state = "cosmohonk"
+	item_state = "cosmohonk"
+	desc = "A lightweight space suit with... HONK"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 20)
```
Comment (optimumtact) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: there's no need to override the armour if it's the same as the parent type.

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -236,6 +236,21 @@ Contains:
...
+	item_state = "cosmohonk"
+	desc = "A lightweight space suit with... HONK"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 20)
+
+/obj/item/clothing/head/helmet/space/eva/clown
+	name = "cosmohonk helmet"
+	icon_state = "cosmohonk"
+	item_state = "cosmohonk"
+	desc = "A lightweight space helmet with... HONK"
+	flash_protect = 0
```
Comment (optimumtact) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: same for the flash protect

**Comment by Fox-McCloud** [comment]
Paradise's suits look better than this, IMO.

**Comment by RandomMarine** [comment]
It looks like the wearer is slouching.

**Comment by bgobandit** [comment]
clown hardsuit 

we becoming nox

**Comment by FantasticFwoosh** [comment]
I didn't know the clown took fashion tips from Hilary Devey

**Comment by Mercenaryblue** [comment]
Pink-Purple? please no

Recolor red/yellow stripped please.

**Comment by Robustin** [comment]
I support clown hardsuit but it needs a different look.

Not just the colors, it needs to have a HARDSUIT look to it, not just "clown thats been bulking" look. The helmet is alright but everything below it has to go. **Try not to go for something that LITERALLY looks like a clown suit and go for a hardsuit that symbolically represents the clown. **

**Comment by GunHog** [comment]
It is _not_ a hardsuit! It is a standard EVA suit styled for the clown. I feel it needs more clowny features. Make slippery (via the `Crossed()` proc), and make it slip the clown on `hit_reaction()'!

It is a clown item! It needs to do something completely pointless and silly!

**Comment by KorPhaeron** [comment]
Security PR has priority

**Comment by KorPhaeron** [comment]
Stale PR try again later if you want

**Pull request closed without merging**